app/assets/stylesheets/base.scss

```diff
@@ -339,7 +339,7 @@ table {
 }
 
 .audits-pagination.content-view-pf-pagination.table-view-pf-pagination {
```

Comment:
Why do we need multiple classNames? the `content-view-pf-pagination.table-view-pf-pagination` is already injected

If we want to keep the borders, I suggest an almost white background:

```
.audits-pagination {
  margin-top: -15px !important;
  border-top: 1px solid #d1d1d1 !important;
  background-color: #fbfbfb;
}
```
Or we can keep the last behaviour of the audits pagination and have no border with white background
```
.audits-pagination {
  margin-top: -15px !important;
  border: none;
  background-color: #fff;
}
```
not sure which way is better

Reply:
@glekner,
Yes, now don't need to add multiple classNames. removing it.